src/modules/toolbox/components/shareToolContent/ShareToolDialog.js

```diff
@@ -51,10 +54,20 @@ export class ShareToolDialog extends BaElement {
     `;
 	}
 
-	async _copyValueToClipboard(value) {
-		await this._shareService.copyToClipboard(value).then(() => { }, () => {
+	/**
+	 * @private
+	 */
+	async _copyValueToClipboard() {
+		const value = 'the link';
+		try {
+			await this._shareService.copyToClipboard(value);
+			emitNotification(`${value} ${this._translationService.translate('map_contextMenuContent_clipboard_success')}`, LevelTypes.INFO);
```

Comment:
Don't use i18n from a foreign module, add your own i18n key/value entry to the i18n of the toolbox module
(see also line 67)